src/test/crossFetchNode.test.ts

```diff
@@ -19,107 +19,10 @@ type TestConfigParameters = typeof config.parameters;
  * Both POST and GET are exercised to ensure compatibility.
  */
 
-const customerClient = new ShopperCustomers(config);
 const searchClient = new ShopperSearch(config);
 
 beforeEach(() => nock.cleanAll());
 
-test('test getting a token with a post operation', async () => {
```

Comment:
This is the old deprecated way to retrieve access tokens. Since we have tests for the slas helpers, I think we can remove these tests that call `authorizeCustomer`